Docs/MANUAL.md

```diff
@@ -0,0 +1,322 @@
+# Gordian Seed Tool Manual
+
+<a href="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-listing.jpeg"><img src="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-listing.jpeg" align="right" width=250 style="border:1px solid black;"></a>
+
+**Gordian Seed Tool** is an iOS-based seed manager that is a reference app for the Gordian system. It allows you to safely and securely store your cryptographic seeds and to export public keys, private keys, and seeds as required.
+
+Why use **Seed Tool**? Because storing your seeds on a fully networked device is a major security vulnerability that also leaves your seeds vulnerable to loss. **Seed Tool** resolves both of these problems. You can move selected public and private keys online only as they're required, and you can be sure that your seeds are in a secure vault that's backed up and not dependent on a single device.
```

Comment:
c/storing you seeds on a fully networked/ storying your seeds IN THE UNENCRYPTED RAM of a fully networked device/

Reply:
Done.